packages/python/patch-no-opendir

```diff
@@ -0,0 +1,36 @@
+--- Modules/posixmodule.c
++++ Modules/posixmodule.c
+@@ -3663,7 +3663,9 @@
+         }
+ 
+         Py_BEGIN_ALLOW_THREADS
++#ifdef HAVE_OPENDIR
+         dirp = opendir(name);
```

Comment:
It is safe, `dirp` is initialised to `NULL`, and next line performs an `if (dirp == NULL)` check